extras/conf/config.toml.sample

```diff
@@ -1,5 +1,5 @@
-gluster-block-cli-path = "gluster-block"
-port = 8081
+gluster-block-cli-path = "/usr/bin/gluster-block"
```

Comment:
`/usr/sbin/gluster-block`  https://github.com/gluster/gluster-block/blob/master/gluster-block.spec.in#L74

Reply:
done